pkg/web/api/v1/prepared.go

```diff
@@ -30,6 +30,7 @@ type Person struct {
 	AddrLine2     string `json:"addr_line_2"`
 	City          string `json:"city"`
 	State         string `json:"state"`
+	PostalCode    string `json:"post_code"`
```

Comment:
I looked at the ivms101 proto file and believe this is correct.

Reply:
Yep - that works!